pkgs/development/libraries/cfitsio/default.nix

```diff
@@ -1,19 +1,32 @@
-{ fetchurl, stdenv }:
+{ fetchurl, stdenv
 
- stdenv.mkDerivation {
-  name = "cfitsio-3.430";
+# Optional dependencies
+, bzip2 ? null }:
```

Comment:
I guess we can build this _with_ bz2 support by default.
Tested locally and there is no noticeable increase in closure size.

Reply:
Ok!